### PR TITLE
Aligned macro name OV_CHECK->OPENVINO_ASSERT

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Fixed macro name

### CODE REVIEW

**Review by tomdol — APPROVED** [review]

**Review by mitruska — APPROVED** [review]

File: ngraph/core/include/openvino/core/check.hpp [hunk 1]
```diff
@@ -91,7 +91,7 @@ class OPENVINO_API CheckFailure : public Exception {
 // Then, we define the macro NODE_VALIDATION_CHECK as follows:
 //
 // #define NODE_VALIDATION_CHECK(node, cond, ...) <backslash>
-//     OV_CHECK_HELPER(::ov::NodeValidationFailure, (node), (cond), ##__VA_ARGS__)
+//     OPENVINO_CHECK_HELPER(::ov::NodeValidationFailure, (node), (cond), ##__VA_ARGS__)
```
Comment (ilya-lavrenov) [inline ngraph/core/include/openvino/core/check.hpp:94]: can we have better name like `OPENVINO_ASSERT`?

File: ngraph/core/include/ngraph/check.hpp [hunk 2]
```diff
@@ -18,11 +18,11 @@ using ov::CheckFailure;
...
 }  // namespace ngraph
 
-#define NGRAPH_CHECK_HELPER2(exc_class, ctx, check, ...) OV_CHECK_HELPER2(exc_class, ctx, check, __VA_ARGS__)
+#define NGRAPH_CHECK_HELPER2(exc_class, ctx, check, ...) OPENVINO_CHECK_HELPER2(exc_class, ctx, check, __VA_ARGS__)
 
-#define NGRAPH_CHECK_HELPER1(exc_class, ctx, check) OV_CHECK_HELPER1(exc_class, ctx, check)
+#define NGRAPH_CHECK_HELPER1(exc_class, ctx, check) OPENVINO_CHECK_HELPER1(exc_class, ctx, check)
 
-#define NGRAPH_CHECK(...) OV_CHECK(__VA_ARGS__)
+#define NGRAPH_CHECK(...) OPENVINO_CHECK(__VA_ARGS__)
```
Comment (ilya-lavrenov) [inline ngraph/core/include/ngraph/check.hpp:25]: do we need a dedicated header with check.hpp? can we put this stuff to core/except.hpp?

Comment (ilyachur) [reply ngraph/core/include/ngraph/check.hpp]: Fixed

File: ngraph/core/include/openvino/core/check.hpp (hunk 1 above)
Comment (ilyachur) [reply ngraph/core/include/openvino/core/check.hpp]: Fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**